### DISCUSSION TITLE
Re-add limit=0 (or equivalent) to disable query pagination in REST and GraphQL APIs.

### DISCUSSION THREAD

**Discussion by dhaub-exelixis** [payloadcms/payload#4126] [Feature Requests & Ideas]
As discussed in this discord thread: https://discord.com/channels/967097582721572934/1172339833381851176, my team and others make use of the now disabled `limit=0` feature to return all query results in one get request. Currently the documentation states that disabling pagination is only possible with the local API: https://payloadcms.com/docs/queries/pagination#disabling-pagination-within-local-api

In the case of my team, we make use of a very high volume of data, that could often have tens of thousands of entries returned in a single query. While we could use a very large number as the limit param, it would be nice to just disable limiting all together.

**Comment by denolfe** [comment]
We're looking into bringing this back. I don't think removing it was intentional either way.

**Reply by denolfe** [reply]
Converted to Issue https://github.com/payloadcms/payload/issues/4143
